src/reporters/plain.ts

```diff
@@ -53,7 +51,6 @@ export function* plainReporter(
 
 	if (configResults.filesResults.size === 0) {
 		yield styleText("green", "No linting issues found.");
-		return;
```

Comment:
🐛 The plain reporter shouldn't skip printing formatting results when there are no lint results.